include/libu64/gfxprint.h

```diff
@@ -2,23 +2,42 @@
 #define LIBU64_GFX_PRINT_H
 
 #include "ultra64.h"
+#include "color.h"
+
+typedef void* (*PrintCallback)(void*, const u8*, s32);
+typedef struct gfxprint {
+    /* 0x00 */ PrintCallback proutFunc; /* Current print out func */
+    /* 0x04 */ Gfx* gListp; /* Current display list to write text to */
+    /* 0x08 */ u16 positionX; /* Display coords (sub-pixel units) */
+    /* 0x0A */ u16 positionY; /* Display coords (sub-pixel units) */
+    /* 0x0C */ u16 offsetX; /* Display offset */
+    /* 0x0E */ u8 offsetY; /* Display offset */
+    /* 0x0F */ u8 flags; /* 1: Kana mode -> 0: Katakana/1: Hiragana
+                 2: Gradient displayed
+                 4: Shadow displayed
+                 8: Attribute changed
+                 64: Low-res -> high-res conversion
+                 128: Opened state */
+    /* 0x10 */ Color_RGBA8 color; /* Current font color */
+    /* 0x14 */ char dummy[0x1C];
+} gfxprint; // size = 0x30
```

Comment:
same here.
Do you even need the members? if you only need the struct to exist then you could use a size placeholder instead